### DISCUSSION TITLE
Transactions API

### DISCUSSION THREAD

**Discussion by bigshooTer39** [ping-pub/explorer#308] [Q&A]
I'd like to setup a Transactions API into Power Query for personal portfolio tracking.  I built a MS Access database which leverages APIs that I setup in Power Query.  I've been able to setup APIs for prices and a few networks but I haven't been able to find anything for Cosmos ecosystem.  RIght now, I have to download everything from stake.tax and import manually.

So, is it possible to use ping.pub to pull transactions via API for my addresses?  I have never used CLI unfortunately.  When I ask about an API, I'm asking if there's endpoint similar to what CoinGecko has for pulling in data on tokens such as historic and current prices.

**Comment by liangping** [comment]
Hey,  thanks for your efforts to improve Ping Dashboard, Ms Access Database would not be a good option for API services, we are open to build everything with community, but stability and reliability is the first thing we have to considering.
